module/support/logging/DataLogging/data/config/DataLogging.yaml

```diff
@@ -1,6 +1,7 @@
 output:
   directory: log
   split_size: 5 * GiB
+  max_size: 10630044057 # 10 gigabytes
```

Comment:
more context. This change doesn't make much sense past the context of this robocup, but future team members are going to think it's here for a reason which is still a thing later